Resources/Prototypes/Roles/Jobs/Security/brigmedic.yml

```
@@ -0,0 +1,53 @@
+
+- type: startingGear
+  id: BrigmedicGear
+  equipment:
+    id: BrigmedicPDA
+    eyes: ClothingEyesBrigMedSec   # Goobstation
+    ears: ClothingHeadsetBrigmedic # Goobstation
+    belt: ClothingBeltBrigmedicWebbingFilled
+    #pocket1: UniqueBrigmedicLockerTeleporter                      # NTR/BSO-like teleporter. 
+  storage:
+    back:
+    - Flash
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)